### ISSUE TITLE
copying ami across regions with the error "missing required parameter: Name"

### ISSUE THREAD

**Issue by henrysher** [mitchellh/packer#2172]
I met this problem using the packer build in **master** branch with the default packer configuration in README:
```json
{
  "builders": [{
    "type": "amazon-ebs",
    "access_key": "XXXXX", 
    "secret_key": "XXXXXXXXXXXXXX",
    "region": "us-east-1",
    "source_ami": "ami-de0d9eb7",
    "instance_type": "t1.micro",
    "ssh_username": "ubuntu",
    "ami_regions": [ "us-east-1", "us-west-2" ],
    "ami_name": "packer-example {{timestamp}}"
  }]
}
```
```
==> amazon-ebs: * Error Copying AMI (ami-6945aa45) to region (us-west-2): InvalidParameter: 1 validation errors:
==> amazon-ebs: - missing required parameter: Name
```
After looking into the [AWS API: CopyImage](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CopyImage.html), it seems "**Name**" parameter now required.
However, the code for ami copy does not take the name parameter: https://github.com/mitchellh/packer/blob/master/builder/amazon/common/step_ami_region_copy.go
```python
	regionconn := ec2.New(awsConfig)
	resp, err := regionconn.CopyImage(&ec2.CopyImageInput{
		SourceRegion:  &source,
		SourceImageID: &imageId,
	})
```

**Comment by henrysher** [comment]
@mitchellh add a new name parameter in user's configuration, or the same name as the original AMI one, what do you prefer?

**Comment by henrysher** [comment]
@mitchellh i prepared a PR and taken the second choice: using the same ami name in the packer configuration.

**Comment by henrysher** [comment]
Fixed in #2178
